containers/Python/.devcontainer/library-scripts/azcli-debian.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+#-------------------------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
+#-------------------------------------------------------------------------------------------------------------
+#
```

Comment:
this library is common across all decontainers, should we package it in a way so they can all reuse without duplicating the code?